catalog/EDF/projects_EDF_MethaneSAT_public-preview_L3concentration.jsonnet

```diff
@@ -0,0 +1,297 @@
+//   - Extensions: https://github.com/stac-extensions/stac-extensions.github.io
+//     - Electro-Optical: https://github.com/stac-extensions/eo
+//     - Synthetic Aperture Radar: https://github.com/stac-extensions/sar
+//     - Scientific: https://github.com/stac-extensions/scientific
+//     - Version: https://github.com/stac-extensions/version
+
+// This jsonnet file has to be named using the following convention:
+//   <subdir>/<id_with_slashes_replaced_with_underbars>.jsonnet.
+
+// The asset id as used in Earth Engine:
+//   ee.ImageCollection('TEMPLATE/IMAGE_COLLECTION_V2_3');
+local id = 'projects/EDF/MethaneSAT/public-preview/L3concentration';
+
+// The directory under 'catalog' that contains the dataset.
+// For datasets under 'projects', leave off the 'projects' component.
+//   E.g., the 'projects/planet-nicfi/assets/basemaps/africa' asset is in the
+//   'planet-nicfi' subdirectory.
+local subdir = 'EDF';
+
+// The version field can be any string. However, it is best to use
+// the exact string that the data provider uses.
+// Do not include a leading `V` in the version string.
+// Prefer Semantic Versioning: https://semver.org/, which uses
+// one to three numeric fields separated by decimal points.
+// The version string might be different than the version portion of the `id`
+// field, as the `id` field cannot have `.` characters.
+local version = '1.0.0';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+local units = import 'units.libsonnet';
+
+// Change this to one of the licenses in:
+// https://github.com/google/earthengine-catalog/blob/main/catalog/spdx.libsonnet
+//
+// - Least restrictive licenses (public domain or atribution-only licenses like
+//   CC-BY-4) are preferred.
+// - If the license is not in spdx.libsonnet, but is available in SPDX, you
+//   can add it to spdx.libsonnet. See: https://spdx.org/licenses/
+// - All other licenses should use spdx.proprietary. Here "proprietary" just
+//   means that the license is not in SPDX.
+// - If multiple licenses apply at the same time, use 'spdx.various'.
+// - If there is a choice of more than one license, pick the most permissive
+//   license.
+// - Non-commercial and share-alike licenses are strongly discouraged, but if
+//   they have to be used, add the dataset to
+//   https://github.com/google/earthengine-catalog/blob/main/non_commercial_datasets.jsonnet
+local license = spdx.proprietary;
+
+// These are helper variables used below. Most files will just leave them as-is.
+local basename = std.strReplace(id, '/', '_');
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+{
+  id: id,
+  // Do not end the title with punctuation. Include version if it is known.
+  title: 'MethaneSAT L3 Concentration Public Preview V' + version,
+  version: version,
+
+  description: |||
+    This early “Public Preview” dataset provides geospatial data for the 
```

Comment:
please use regular and not curly quotes

Reply:
done.  moved files to new location, will resolve this and we can continue over there.